src/governance/ProfitManager.sol

```diff
@@ -183,6 +205,17 @@ contract ProfitManager is CoreRef {
         emit SurplusBufferUpdate(block.timestamp, newSurplusBuffer);
     }
 
+    /// @notice withdraw from surplus buffer of a given term
+    function withdrawFromTermSurplusBuffer(
+        address term,
+        uint256 amount
```

Comment:
adding the to parameter here would save you an extra external call in SurplusGuildMinter

Reply:
Done in [9958e17](https://github.com/volt-protocol/ethereum-credit-guild/pull/31/commits/9958e1741b52a43f7b5548e7353e260d4ca3d2db)